desc/examples/update_all_equilibria_backwards_compatibility.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+"""
+Run this script to update all the DESC equilibria in desc/examples.
+
+If you added a new IO attribute to the DESC equilibrium, this code
+will update output files to include the new attribute. Please make sure
+you add special handler to eq._set_up() function for new attribute. By default
+the new attribute will be set to None.
+
+python3 update_all_equilibria_backwards_compatibility.py
+
+Copy the two lines below to the main code to run it on a GPU
+from desc import set_device
+set_device("gpu")
+"""
+
+import glob
+import os
+
+from desc.io import load
+
+pwd = os.getcwd()
+
+for fname in glob.glob(pwd + "/*.h5"):
+    foutputname = fname.split(".")[0].split("/")[-1]
+    print(f"\nUpdating the output file {foutputname} \n")
+    # load old output file
+    eqfam = load(fname)
+    print(eqfam)
+    for i, eq in enumerate(eqfam):
+        print(eq)
+        # update the output file
+        eq._set_up()
```

Comment:
`_set_up()` automatically gets called after loading an object so you don't need to call it explicitly here

Reply:
@f0uriest About this script, I have a question. Some of the tests expect some old files to have lacking attributes, and use
```python
with pytest.warns(RuntimeWarning):
    eq = load(desc_no_current_path)[-1]
```
type of loading. but since the new script adds every attribute with a default value, there is no RuntimeWarning thrown and pytest complains. Should I add exception for old files, so that it won't update them or change the tests?

Reply:
Ahh yeah we should add an exception for those, or possibly remove those tests if we're confident they're not needed. Should check with @dpanici and @ddudt